### DISCUSSION TITLE
Select button color recommendation

### DISCUSSION THREAD

**Discussion by Kim-Jiyun** [AgainIoT/Open-Set-Go#56] [Feature Suggestion / Share Ideas]
![license](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/25cf8d91-5d5a-4914-83ec-52d80c91f99e)
I'm thinking about the color of the select button on the license page shown in the picture and posting it to get a recommendation!
사진에 보이는 license 페이지에 select 버튼 색깔을 고민하다가 추천받기 위해서 글을 올립니다!

![select](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/67b8c4cb-2da9-4e0e-b849-5b576503385f)
Select, that is, the color before selection, I referred to many other sites and found that most of them were gray, so I thought it would be better for us to do gray. So, Selected, that is, I prepared 6 different versions of the color when I selected it. We tried to give uniformity by choosing all the colors that we use in pygma. The purple color is the logo color, the blue color is the next button at the bottom and the unity with our entire screen, the red color is easily visible, and the green color is the existing color.
Select, 즉 선택하기 전 색깔은 많은 다른 사이트들을 참고한 결과 대부분이 회색으로 하여 우리도 회색으로 하는게 좋겠다고 생각했습니다. 그래서 Selected, 즉 선택했을 때에 색깔을 6가지 버전으로 준비했습니다. 후보 색깔은 모두 피그마에 있는 우리가 사용하는 색깔로 골라서 통일성을 주려고 했습니다. 보라색은 로고 색깔, 푸른 계열은 밑에 next 버튼과 우리 전체 화면과의 통일성, 빨강 계열은 눈에 잘 띄는 색깔, 초록은 기존 색깔로 정리했습니다.

If you have any other opinions, feel free to let me know!
혹시, 또 다른 의견이 있다면 자유롭게 알려주세요!

![nextbutton](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/d028ef5f-e42c-4893-a7d4-b4dfe6e4712a)
I also attach a picture of the next button for your reference.
참고를 위해 next 버튼 사진도 첨부합니다.

**Comment by ymw0407** [comment]
I think the existing green/grey combination is better.
전 기존의 초록색/회색이 더 나은 조합 같습니다.

**Comment by ancy0** [comment]
For the unselected condition, I think a gray color with a bluer light would be better than the gray in the example above. And I think navy or green would be good for the selected state. 
저는 선택되지 않은 상태의 경우 위의 예시의 회색보다 더 푸른 빛을 띠는 회색 색상이 좋을 것 같습니다. 그리고 선택한 상태의 경우 남색이나 초록색이 좋을 것 같습니다. 

Gray is mostly used when disabled. 
You probably need to select colors for clickable and clicked states, but if you apply gray buttons to unselected items, you should also think about the colors that will appear in the hover effect. If there is no effect, it is difficult to know if the button is clickable.
회색은 대부분 비활성화 상태일 때 주로 사용됩니다. 
아마 클릭 가능한 상태와 클릭된 상태에 대한 색상 선택이 필요할텐데 만약 선택되지 않은 아이템에 대해서 회색 버튼을 적용한다면 hover 효과 시 나타날 색상도 생각해봐야할 것 같습니다. 아무런 효과가 없다면 해당 버튼이 클릭 가능한 상태인지 알기 어렵기 때문입니다.

Depending on the color of the button, it would be good to consider the color of the text and the border.
버튼 색상에 따라서 텍스트 색상 및 테두리 여부도 고려하면 좋을 것 같아요.

**Comment by hummingbbird** [comment]
I also don't think it's good to use gray buttons that aren't selected. This is because gray is usually disabled, meaning it is not pressed.
I'm going to come up with these ideas.
저도 선택되지 않은 버튼을 회색으로 하는 건 좋지 않다고 생각합니다. 보통 회색의 경우 비활성화 즉, 눌리지 않는다는 것을 의미하기 때문입니다.
이런 아이디어를 제시합니다.

1.
* select (unselected) -> Light blue
* Selected (only selected items) -> navy

2.
* select (unselected) -> Light green (similar to white)
* selected (selected items only) -> green

**Comment by Kim-Jiyun** [comment]
Thank you for your various comments!
다양한 의견 감사합니다!

In summary, we decided to fill in green when it is selected with green fringes when it is selected.
의견을 종합해 select(unselect)일 때는 초록색테두리로 selected 일때는 초록색을 채우는 것으로 결정했습니다. 
![select](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/2509ad3e-a03b-466e-8eb5-335a46e6eb82)
![selected](https://github.com/AgainIoT/Open-Set-Go/assets/84488029/bd701e7a-0287-4370-9ca6-bbe4321aafbf)

**Comment by ymw0407** [comment]
This discussion resolved at AgainIoT/Open-Set-Go_client#159 !!!
